### PR TITLE
Attempt to fix #5450

### LINKED ISSUE #5450: Show start time on query execution results
After query execution (MySQL), it shows a tab with "Statistics" or results (updated rows, query, finish time). Would be great if we could also see the **start time** of the query. It would be also cool to see elapsed time.

### PR DESCRIPTION
**Pull request by bob27aggiustatutto**
Attempt to fix #5450

### CODE REVIEW

**Comment by E1izabeth** [comment]
Thank you for contribution.
Maybe I didn't notice, please guide me where the start time is recorded?

**Comment by bob27aggiustatutto** [comment]
into `org.jkiss.dbeaver.model.exec.DBCStatistics` in the `long startTime` field
https://github.com/dbeaver/dbeaver/blob/devel/plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/exec/DBCStatistics.java#L29
@E1izabeth

**Comment by E1izabeth** [comment]
Yes, but where is the value of this field set?

**Comment by bob27aggiustatutto** [comment]
I don't think I understand your question, maybe here is the answer
https://github.com/dbeaver/dbeaver/blob/devel/plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/exec/DBCStatistics.java#L42
@E1izabeth

**Comment by E1izabeth** [comment]
Yes, that's what I missed. Thank you.

File: plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/exec/DBCStatistics.java [hunk 1]
```diff
@@ -104,6 +104,10 @@ public long getEndTime() {
         return startTime + getTotalTime();
     }
 
+    public long getStartTime (){
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/exec/DBCStatistics.java]: Could you add a whitespace here?
```suggestion
    public long getStartTime () {

```

**Review by E1izabeth — APPROVED** [review]

File: plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/exec/DBCStatistics.java [hunk 2]
```diff
@@ -104,6 +104,11 @@ public long getEndTime() {
         return startTime + getTotalTime();
     }
 
+    public long getStartTime () {
+
+        return this.startTime;
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/exec/DBCStatistics.java]:
```suggestion
    public long getStartTime() {
        return this.startTime;

```

**Review by ShadelessFox — APPROVED** [review]

**Review by ShadelessFox — APPROVED** [review]

**Review by LonwoLonwo — APPROVED** [review]

**Comment by Matvey16** [comment]
Verified

**Pull request merged**